benchmarks/src/main/scala/zio/redis/BenchmarksUtils.scala

```diff
@@ -17,22 +17,24 @@
 package zio.redis
 
 import cats.effect.{IO => CatsIO}
-
+import zio.{BootstrapRuntime, ZIO, ZLayer}
 import zio.logging.Logging
 import zio.redis.codec.StringUtf8Codec
 import zio.schema.codec.Codec
-import zio.{BootstrapRuntime, ZIO, ZLayer}
 
 trait BenchmarksUtils {
   self: RedisClients with BootstrapRuntime =>
 
   def unsafeRun[CL](f: CL => CatsIO[Unit])(implicit unsafeRunner: QueryUnsafeRunner[CL]): Unit =
     unsafeRunner.unsafeRun(f)
 
-  def zioUnsafeRun(source: ZIO[RedisExecutor, RedisError, Unit]): Unit =
+  def zioUnsafeRun(source: ZIO[Redis, RedisError, Unit]): Unit =
     unsafeRun(source.provideLayer(BenchmarksUtils.Layer))
 }
 
 object BenchmarksUtils {
-  private final val Layer = Logging.ignore ++ ZLayer.succeed[Codec](StringUtf8Codec) >>> RedisExecutor.local.orDie
+  private final val Layer = {
+    val executor = Logging.ignore >>> RedisExecutor.local.orDie
+    executor ++ ZLayer.succeed[Codec](StringUtf8Codec) >>> Redis.live
```

Comment:
Just a heads up, once this PR lands, I'll submit the removal of `StringUtf8Codec`.